### DISCUSSION TITLE
[Linux] Wrong Resolution when starting the game or maximing it after being minimised

### DISCUSSION THREAD

**Discussion by Aurvandill** [ppy/osu#18403] [Q&A] [answered]
The Game starts with a weird resoltuion even tho 1920x1080p is selectected
changing the resolution back and forth fixes this problem.
after minimising the game and opening it back up again it has the same problem

i tried runnign osu in borderless and fullscreen mode and both have the same issue

some more Info
OS: Fedora 36
Display Server: Wayland
Window Manager: Gnome 42
osu Version: 2022.523.0-lazer

Screenshot
![Bildschirmfoto vom 2022-05-24 19-17-40](https://user-images.githubusercontent.com/22366193/170095734-08f02fca-c98e-4f54-8795-2c9e196c1f2a.png)

Logs:
[database.log](https://github.com/ppy/osu/files/8764940/database.log)
[runtime.log](https://github.com/ppy/osu/files/8764937/runtime.log)
[performance.log](https://github.com/ppy/osu/files/8764938/performance.log)
[network.log](https://github.com/ppy/osu/files/8764939/network.log)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/18301

**Reply by Aurvandill** [reply]
seems to be fixed for me in the newest (2022.528.00) update of osu lazer
